src/Initializer/InitialFieldProjection.cpp

```diff
@@ -120,4 +122,9 @@ void seissol::initializers::projectInitialField(std::vector<std::unique_ptr<phys
 #ifdef _OPENMP
   }
 #endif
+
+  seissol::initializer::synchronize(lts.dofs);
```

Comment:
Isn't it a bit strange to have this call in a file called InitialFieldProjection.cpp?
(I mean this has nothing to do with field projection?

Reply:
It kind of has something to do with it—as the purpose of the method is to set the DOFs to the initial condition data. That should also includes the duplicates in the LTS tree.

(it's the same in the end as if we were to just iterate over all non-ghost layers in the LTS tree—except that we do not compute the initial condition multiple times for each duplicate)

Reply:
I've removed the calls in `InitModel`; the duplicates were really initialized before that already, because we went through the whole LTS tree there.